PHP/PHP Source.sublime-syntax

```diff
@@ -589,12 +592,59 @@ contexts:
         - include: identifier-constant-pop
     - include: use-statement-common
     - include: identifier-constant-pop
+  enum:
+    - match: '(?i)\b(enum)\s+({{identifier}})\s*'
+      captures:
+        1: keyword.declaration.enum.php
+        2: entity.name.enum.php
+      push: enum-definition
+
+  enum-definition:
+    - match: "(?=;)"
```

Comment:
Are the double quotes necessary here?

Reply:
I guess this is a general issue in several locations of this and other syntaxes.